prediction_market_agent_tooling/markets/agent_market.py

```
@@ -301,16 +349,19 @@ def get_outcome_index(self, outcome: str) -> int:
-    def get_position(self, user_id: str) -> Position | None:
+    def get_position(self, user_id: str) -> ExistingPosition | None:
         raise NotImplementedError("Subclasses must implement this method")
 
     @classmethod
     def get_positions(
-        cls, user_id: str, liquid_only: bool = False, larger_than: float = 0
-    ) -> list[Position]:
+        cls,
+        user_id: str,
+        liquid_only: bool = False,
+        larger_than: OutcomeToken = OutcomeToken(0),
```

Ruff (0.8.2)

Do not perform function call `OutcomeToken` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)